evm_loader/program/src/error.rs

```diff
@@ -0,0 +1,28 @@
+//! Error types
+
+use num_derive::FromPrimitive;
+use solana_program::{decode_error::DecodeError, program_error::ProgramError};
+use thiserror::Error;
+
+/// Errors that may be returned by the EVM Loader program.
+#[derive(Clone, Debug, Eq, Error, FromPrimitive, PartialEq)]
+pub enum EVMLoaderError {
+    /// Storage Account is uninitialized.
+    #[error("Storage Account is uninitialized")]
+    StorageAccountUninitialized,
+    /// SomeError.
+    #[error("Some error")]
+    SomeError,
+}
+
+impl From<EVMLoaderError> for ProgramError {
+    fn from(e: EVMLoaderError) -> Self {
+        ProgramError::Custom(0xff00ff00 + (e as u32))
```

Comment:
By changing "0xff00ff00" part we can define our errors. without it error starts from 0x
"0xff00ff00" - just for example
Here example for error with "0xff00"
 {'code': -32002, 'message': 'Transaction simulation failed: Error processing Instruction 0: custom program error: 0xff00', 'data': {'err': {'InstructionError': [0, {'Custom': 65280}]}, 'logs': ['Program 8bSfmpJhTXYz1ng7mecKtDrA9UZkTeJqRZt9v4NGHb1M invoke [1]', 'Program log: Instruction parsed', 'Program 8bSfmpJhTXYz1ng7mecKtDrA9UZkTeJqRZt9v4NGHb1M consumed 2312 of 10000000 compute units', 'Program 8bSfmpJhTXYz1ng7mecKtDrA9UZkTeJqRZt9v4NGHb1M failed: custom program error: 0xff00']}}

Reply:
Why we can't start our errors from 0x0? As I understand, we get a `custom program error` with code defined inside the contract.
If we first `SomeError` as the first element in EVMLoaderError, 0x0 will match it. And other errors will have non-empty codes.